subprojects/core/src/integTest/groovy/org/gradle/initialization/CalculateTaskGraphBuildOperationIntegrationTest.groovy

```diff
@@ -83,6 +83,38 @@ class CalculateTaskGraphBuildOperationIntegrationTest extends AbstractIntegratio
         operation().failure.contains("Task 'someNonExisting' not found in root project")
     }
 
+    def "build path for calculated task graph is exposed"() {
+        settingsFile << """
+            includeBuild "b"
+        """
+
+        buildFile << """
+            apply plugin:'java'
+            
+            dependencies {
+                compile "org.acme:b:1.0"            
+            }
+        """
+
+        file("b/build.gradle") << """
+            apply plugin:'java'
+            group = 'org.acme'
+            version = '1.0'
+        """
+        file('b/settings.gradle') << ""
+
+        when:
+        succeeds('build')
+
+        def taskGraphCalculations = buildOperations.all(CalculateTaskGraphBuildOperationType)
```

Comment:
Just checking: is the ordering here guaranteed?

Reply:
in the test it should be yes.